mteb/tasks/BitextMining/multilingual/RomaTalesBitextMining.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskBitextMining
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class RomaTalesBitextMining(AbsTaskBitextMining):
+    metadata = TaskMetadata(
+        name="RomaTalesBitextMining",
+        dataset={
+            "path": "kardosdrur/roma-tales",
+            "revision": "f4394dbca6845743cd33eba77431767b232ef489",
+        },
+        description="Parallel corpus of Roma Tales in Lovari with Hungarian translations.",
+        reference="https://idoc.pub/documents/idocpub-zpnxm9g35ylv",
+        type="BitextMining",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["rom-Latn", "hun-Latn"],
+        main_score="f1",
+        date=None,  # Unknown, these are folk tales
```

Comment:
Can we do like a very rough time span?

Reply:
I think some of them might originate from before Christ, it wouldn't make a lot of sense xd